script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -563,7 +564,8 @@ A Mystic Code originating in death by weakness, or death by torture.
 [sha:97a6ca1b80d5b029491b4a35f70b5c0c46baa185]{
 -- Page 614, Offset 12466.
 -- 使用者の魔力が尽きぬかぎり、死して魔女と証明されないかぎり、最低限の生命活動を保証する聖女の<潔|まも>り―――！
-As long as the user hasn't depleted her magical energy and hasn't died being proven to be a witch, the <purity|p r o t e c t i o n> of the Holy Maiden will provide the bare minimum needed to ensure survival―――!
+// umineko reference or something also im about to kill myself
+As long as the user hasn't depleted her magical energy or hasn't been executed as a witch, the <purity|p r o t e c t i o n> of the Holy Maiden will provide the bare minimum needed to ensure survival―――!
```

Comment:
is this translated correctly? Wouldn't this be more like "as long as not a dead witch [and] proven [as a witch]"?
>  死して魔女と証明されないかぎり